language/stdlib/modules/designated_dealer.move

```diff
@@ -0,0 +1,150 @@
+address 0x0 {
+    module DesignatedDealer {
+        use 0x0::LibraTimestamp;
+        use 0x0::Vector;
+        use 0x0::Transaction as Txn;
+        use 0x0::Libra;
+        use 0x0::LibraAccount;
+        use 0x0::Coin1;
+        use 0x0::Coin2;
+
+        resource struct Dealer {
+            // Time window start in microseconds
```

Comment:
Using `///`  comments here and on functions will augment the generated markdown documentation (if desired)